packages/contracts-bedrock/src/L1/StandardValidator.sol

```diff
@@ -513,23 +513,19 @@ contract StandardValidatorBase {
         string memory _errors,
         IDisputeGameFactory _dgf,
         IAnchorStateRegistry _asr,
-        IProxyAdmin _admin,
+        IProxyAdmin,
         GameType _gameType,
         string memory _errorPrefix
     )
         internal
         view
+        virtual
         returns (string memory)
     {
         _errorPrefix = string.concat(_errorPrefix, "-ANCHORP");
         _errors = internalRequire(
             stringEq(_asr.version(), anchorStateRegistryVersion()), string.concat(_errorPrefix, "-10"), _errors
         );
-        _errors = internalRequire(
-            _admin.getProxyImplementation(address(_asr)) == anchorStateRegistryImpl,
-            string.concat(_errorPrefix, "-20"),
-            _errors
-        );
```

Comment:
Why do we no longer want this check in the `StandardValidatorBase` contract?

Reply:
V180 of the ASR contains immutables, so we can't validate the implementation. Immutables were removed in V200.